### ISSUE TITLE
Loss during self-supervised training stage

### ISSUE THREAD

**Issue by kuwar81523** [emadeldeen24/TS-TCC#3]
During the self-supervised training stage the training loss is not decreasing much it's hovering around 10. But in the fine tuning stage we are getting good accuracy. Is there any prominence to the self supervised training loss and should we even consider looking at the self supervised training loss values?

**Comment by emadeldeen24** [comment]
I can't generalize, but for me, I used not to consider the SSL loss as a quality measure, but the linear evaluation or the fine-tuning as you mentioned .. this also applies for some other methods I experienced, not only TS-TCC.

**Comment by kuwar81523** [comment]
ok thanks for clarifying
